client/hosting/overview/components/plan-bandwidth.tsx

```diff
@@ -22,9 +36,22 @@ const getCurrentMonthRangeTimestamps = () => {
 };
 
 export function PlanBandwidth( { siteId }: PlanBandwidthProps ) {
+	const [ bandwidthData, setBandwidthData ] = useState< SiteMetricsAPIResponse >();
 	const selectedSiteData = useSelector( getSelectedSite );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+	const isAtomic = useSelector( ( state ) => isAtomicSite( state, siteId ) );
+	const canViewStat = useSelector( ( state ) => canCurrentUser( state, siteId, 'publish_posts' ) );
+	const { isSiteAtomic, hasSftpFeature } = useSelector( ( state ) => ( {
+		isSiteAtomic: isSiteWpcomAtomic( state, siteId as number ),
```

Comment:
What's the difference between `isAtomicSite` and `isSiteWpcomAtomic`? AFAICT, they're retrieved from different API endpoints, but I assume they indicate the same thing. 

Is there a difference?